geometric_kernels/spaces/product.py

```diff
@@ -260,7 +260,7 @@ def dim_of_eigenspaces(self):
 
 
 class ProductDiscreteSpectrumSpace(DiscreteSpectrumSpace):
-    def __init__(self, *spaces: DiscreteSpectrumSpace, num_eigen: int = 100):
+    def __init__(self, *spaces: DiscreteSpectrumSpace, num_eigenvalues: int = 32):
```

Comment:
which ones, unique, with repetitions, or is this actually the number of levels (which is, in general, neither of these)?